test/NuGet.Core.Tests/NuGet.Credentials.Test/PluginCredentialProviderTests.cs

```diff
@@ -569,10 +571,14 @@ public async Task OnPluginUnexpectedStatusException_NoRetryIfVerbosityFlagWasNot
         [Fact]
         public void SetsIdBasedOnTypeAndFilename()
         {
+            FileInfo providerFileInfo = RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+                ? new FileInfo(@"c:\some\path\provider.exe")
+                : new FileInfo("/some/path/provider");
+
             var mockLogger = new Mock<Common.ILogger>();
-            var provider = new PluginCredentialProvider(mockLogger.Object, @"c:\some\path\provider.exe", 5, "Normal");
+            var provider = new PluginCredentialProvider(mockLogger.Object, providerFileInfo.FullName, 5, "Normal");
```

Comment:
This makes the test pass on non-Windows so its now being run on other platforms.